### PR TITLE
models/public: move to https://storage.openvinotoolkit.org/repositories/open_model_zoo/public/2022.1

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

File: models/public/alexnet/model.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ files:
   - name: alexnet.caffemodel
     size: 243862414
     checksum: f102235bc9d8f422a04d5254a21fd1862986eae91c7337714fd5d78dcf37f835d05ed7943e80056ae19ca9e9c054c181
-    source: http://dl.caffe.berkeleyvision.org/bvlc_alexnet.caffemodel
```
Comment (alalek) [inline models/public/alexnet/model.yml:35]: It makes sense to keep original URL somewhere, e.g. in `description` field.

Comment (vladimir-dudnik) [reply models/public/alexnet/model.yml:35]: actually, I agree. @Wovchena @jkamelin can we replace source keyword with new, original_source keyword. So, when we upload public models package to storage.openvinotoolkit.org, CI will add us keywords source for actual link where we will download model by default, and original_source will be kept in model.yml files for reference. And potentially, we even may add option to downloader to get models from original links ?

Comment (Wovchena) [reply models/public/alexnet/model.yml:35]: The original URL is kept in `source.txt` similar to how it was done for 2021.4 public models

Comment (vladimir-dudnik) [reply models/public/alexnet/model.yml:35]: yeah, but having this as a keyword in model.yml allows to use it for downloading as an option. and this way we keep all info in single place

**Review by vladimir-dudnik — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
hash mismatch for mtcnn models?

File: models/public/Sphereface/model.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019 Intel Corporation
+# Copyright (c) 2021 Intel Corporation
```
Comment (eaidova) [inline models/public/Sphereface/model.yml:1]: possibly, it should be 2022 :)

**Comment by eaidova** [comment]
@Wovchena @vladimir-dudnik what about updating CONTRIBUTING.md file with info, how public models should be contributed according to this changes now (it may confuse some external contributors...)?

**Comment by Wovchena** [comment]
@jkamelin, can you help me with the error? The clue I have is that concatenation of .prototxt with itself gives the actual hash.

**Comment by Wovchena** [comment]
@jkamelin also running with `use_downloader_cache` results in printing the error twice: http://213.221.44.203/job/omz/job/precommit/job/downloader_ubuntu18/1491/console

**Comment by jkamelin** [comment]
> @jkamelin also running with `use_downloader_cache` results in printing the error twice: http://213.221.44.203/job/omz/job/precommit/job/downloader_ubuntu18/1491/console

This happens because mtcnn and mtcnn-o, mtcnn-p, mtcnn-r are in [tested_models.lst](http://213.221.44.203/job/omz/job/precommit/job/downloader_ubuntu18/1493/artifact/tested_models.lst/*view*/), so when downloading the second time, all files are written to the end of previously downloaded ones. I'll create a fix on downloader side.

**Comment by Wovchena** [comment]
Jenkins please retry a build

File: models/public/alexnet/model.yml (hunk 1 above)
Comment (Wovchena) [reply models/public/alexnet/model.yml:35]: Done

File: models/public/Sphereface/model.yml (hunk 2 above)
Comment (Wovchena) [reply models/public/Sphereface/model.yml]: Updated

**Review by eaidova — APPROVED** [review]

**Pull request merged**